IBPSA/Utilities/IO/Reports/CSVWriter.mo

```diff
@@ -5,37 +5,59 @@ model CSVWriter "Model for writing results to a .csv file"
   parameter Integer nin "Number of inputs"
     annotation(Evaluate=true, Dialog(connectorSizing=true));
   parameter String fileName = getInstanceName() + ".csv" "File name, including extension";
-  parameter String delimiter = "\t" "Delimiter for csv file"
-    annotation(Dialog(group="Options"));
   parameter Modelica.SIunits.Time samplePeriod "Sample period";
+  parameter String delimiter = "\t" "Delimiter for csv file"
+    annotation(Dialog(tab="Advanced"));
+
   parameter Boolean writeHeader = true
     "=true, to specify header names, otherwise no header"
-    annotation(Dialog(group="Options"));
+    annotation(Dialog(tab="Advanced"));
+  parameter Boolean writeCombiTimeTableHeader=true
```

Comment:
Would it be clearer for the user to make a separate block that is a `CombiTimeTableWriter` as opposed to combing csv with the combi time table format?

Reply:
In general, I have a personal preference of having many options instead of many models, since it is easier to avoid code duplication and to test the influence of an option. If you think that we should split it, then I'll split it :)

Reply:
I think in this case we should split it as I don't think it is clear to most users what a CSV file with a combi time table header is. E.g., will Python or Excel still be able to read the CSV file if there is a header for a combi time table? I think it will, but this is probably not clear to most users.

Reply:
ok, will implement two models